src/services/BreadCrumbsConst.js

```diff
@@ -0,0 +1,32 @@
+export const ESPLORA = [{ label: "Home", link: "", id: 1 }];
+export const PROGETTO = [
+  { label: "Home", link: "/", id: 2 },
+  { label: "Il progetto", link: "", id: 3 },
+];
+export const FAQ = [
+  { label: "Home", link: "/", id: 4 },
+  { label: "Domande frequenti", link: "", id: 5 },
+];
+export const RICERCA = [
+  { label: "Home", link: "/", id: 6 },
+  { label: "Ricerca nel catalogo", link: "", id: 7 },
+];
+export const RICERCADETTAGLIO = [
+  { label: "Home", link: "/", id: 8 },
+  { label: "Ricerca nel catalogo", link: "back", id: 9 },
+  { label: "Dettaglio asset", link: "", id: 10 },
+];
+export const PRIVACY = [
+  { label: "Home", link: "/", id: 11 },
+  { label: "Privacy policy", link: "", id: 12 },
+];
+export const BREADCRUMBS = {
+  ESPLORA: ESPLORA,
```

Comment:
I nomi delle variabili possiamo tenerli in inglese? I testi invece sono giusti in italiano.